docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -36,9 +36,11 @@ weights at index time, and to search against later.
 To use ELSER, you must have the {subscriptions}[appropriate subscription] level 
 for semantic search or the trial period activated.
 
-NOTE: The minimum dedicated ML node size for deploying and using 
-the ELSER model is 4 GB. This is a minimum and better performance
-can be achieved by using bigger ML nodes.
+NOTE: The minimum dedicated ML node size for deploying and using the ELSER model 
+is 4 GB in Elasticsearch Service with 
+{cloud}/ec-autoscaling.html[deployment autoscaling] turned on. Autoscaling 
```

Comment:
```suggestion
{cloud}/ec-autoscaling.html[deployment autoscaling] turned off. Autoscaling is recommended because it
```

Reply:
I got a bit confused here. Don't we recommend 4GB **and** Autoscaling turned **on**?

Reply:
Ok, now I got it. You mean "IF you have autoscaling turned OFF, then you should use min 4GB. If you have autoscaling ON, then it takes care of node sizing itself, so don't worry"